core/domain/exp_domain_test.py

```diff
@@ -629,16 +635,37 @@ def test_validation(self):
         init_state.update_interaction_default_outcome(default_outcome.to_dict())
         exploration.validate()
 
+        interaction.solution = state_domain.Solution.from_dict(
+            interaction.id, {
+                'answer_is_exclusive': True,
+                'correct_answer': 'hello_world!',
+                'explanation': {
+                    'content_id': 'solution',
+                    'html': 'hello_world is a string'
+                }
+            }
+        )
+        self._assert_validation_error(
+            exploration,
+            re.escape('Hint(s) must be specified if solution is specified'))
```

Comment:
Any specific reason for using `re.escape` here?

Reply:
The brackets cause problems in assertraisesregexp, thats why I need to escape them